client/src/css_mobile/Navigation.css

```diff
@@ -7,15 +7,49 @@
   transition: top 0.25s;
 }
 
+.nav-topbar-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.nav-mobile-logo {
+  margin: none;
+  padding: none;
+  background-image: url('/asset/icon/logo192.png');
+  background-position: center;
+  background-size: cover;
+  height: 2em;
+  width: 2em;
+  margin-left: 4px;
+}
+
+.nav-mobile-menu {
+  padding-top: 1em;
+}
+
+.nav-mobile-menu-item {
+  background-color: white !important;
+}
+
+.nav-mobile-menu-item.ant-menu-item-selected {
+  font-weight: bold;
+}
+
+.ant-menu-inline .ant-menu-item::after {
+  border-right: 0px none white !important;
+}
+
 .nav-mobile-option {
-  font-size: 2em;
-  font-family: sans-serif;
-  color: #232323;
+  margin-left: 2em;
+
+  color: #232323 !important;
   transition: color 0.3s ease;
-  list-style-type: none;
+  font-size: 1.2em !important;
+  text-align: left !important;
 }
 
 .nav-mobile-option:hover {
-  color: tomato;
-  text-decoration: none;
+  color: tomato !important;
```

Comment:
Can we make the styling similar to the desktop navbar, where if you hover it it's a bolded black, and when its selected it's bolded black as well?